python/doc/source/api.rst

```diff
@@ -167,6 +167,7 @@ Input / Output and Shared Memory
    memory_map
    create_memory_map
    PythonFile
+   feather
```

Comment:
I think you will have to list the particular Feather symbols here. May require using the `.. currentmodule::` directive. I'm working on a patch to include documentation for some other recently-added serialization functions so I can take care of this there